operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys_on_PIT_Node.md

```diff
@@ -1,10 +1,10 @@
-# Set NCN Image Root Password, SSH Keys, and timezone (optional) on PIT Node
+# Set NCN Image Root Password, SSH Keys, and timezone on PIT Node
 
 Modify the NCN images by setting the root password and adding SSH keys for the root account.
-Changing the timezone for the NCNs can also be done at this time. This procedure shows this process being
+Optionally, changing the timezone for the NCNs can also be done at this time. This procedure shows this process being
```

Comment:
I think we want to say that this is not recommended as we have updated other parts of the documentation to prevent people from changing the timezone.

Reply:
I don't disagree per se, but it was a specific request from @haroldlongley, IIRC.